source/_integrations/monzo.markdown

```diff
@@ -0,0 +1,94 @@
+---
+title: Monzo
+description: Instructions on how to integrate Monzo integration into Home Assistant.
+ha_category:
+  - Sensor
+ha_release: "2023.10.1"
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@JakeMartin-ICL'
+ha_config_flow: true
+ha_domain: monzo
+ha_homekit: true
+ha_platforms:
+  - sensor
+related:
+  - docs: /docs/configuration/remote/
+    title: Remote access to Home Assistant
+---
+
+The **Monzo** {% term integration %} allows you to connect your Monzo bank accounts to Home Assistant
+
+- [Prerequisites and Approval](#prerequisites-and-approval)
+  - [Adding a second account](#adding-a-second-account)
+- [Sensor](#sensor)
+- [Webhooks and Triggers](#webhooks-and-triggers)
+- [Services](#services)
+  - [Pot transfer](#pot-transfer)
+  - [(Un-)Register Webhooks](#un-register-webhooks)
+
+{% include integrations/config_flow.md %}
+
+## Prerequisites and approval
+
+1. Before adding the Monzo integration, you'll need to create a [Monzo developer account](https://developers.monzo.com/). 
+2. From here, you need to create a new OAuth client for Home Assistant to use by going to **Clients** > **New OAuth Client**. 
+3. Then, fill in the form as follows:
+
+  - Name: Home Assistant
+  - Logo URL: This can be left blank
+  - Redirect URLs: <https://my.home-assistant.io/redirect/oauth>
+  - Description: For example: Used by the Monzo Home Assistant Integration
+  - Confidentiality: Confidential
+
+4. Make sure to **copy the URL shown** - don't replace it with your own URL.
+5. Once submitted, you can proceed with adding the integration.
+    - Go to {% my integrations title="**Settings** > **Devices & services**" %}, and add the **Monzo** integration.
+    - Fill in the OAuth details for the client you've created in the Monzo developer portal.
+    - **Important** - After authorizing Home Assistant access via email, for security you'll also need to verify again from within the Monzo app. 
+      - A reminder to do this will be displayed in Home Assistant before completing the installation - don't proceed until you've done this from the popup in the mobile app.
+      - If you've forgotten to do this, the integration will fail to load, but you can simply accept the popup and reload the integration without entering your details again.
```

Comment:
```suggestion
5. Once submitted, you can proceed with adding the integration.
   - Go to {% my integrations title="**Settings** > **Devices & services**" %}, and add the **Monzo** integration.
   - Fill in the OAuth details for the client you've created in the Monzo developer portal.
   - **Important** - After authorizing Home Assistant access via email, for security you'll also need to verify again from within the Monzo app. 
     - A reminder to do this will be displayed in Home Assistant before completing the installation - don't proceed until you've done this from the popup in the mobile app.
     - If you've forgotten to do this, the integration will fail to load, but you can simply accept the popup and reload the integration without entering your details again.
```